internals/reminders.py

```diff
@@ -54,18 +72,24 @@ def build_email_tasks(
     # on old Feature entity milestone fields and will need to be refactored
     # to use Stage entity fields before removing this Feature use.
     feature = Feature.get_by_id(fe.key.integer_id())
+    # Check if this notification should be escalated.
+    is_escalated = escalation_check(fe)
```

Comment:
Now that you have a method defined in the base class and overridden in the subclass, you don't need to pass `esclation_check`, instead you can just call `self.should_escalate_notification`.

Reply:
This is happening in `build_email_tasks()`, which is a separate function and not a class method - so we would either have to pass the check function in or pass in the handler object (but I might be missing something)

Reply:
Oops.  Nevermind that part.